langchain/document_loaders/utils/filetype.py

```diff
@@ -0,0 +1,55 @@
+import os
```

Comment:
Admittedly, this is a very naive mime type checker. Not as robust as the one Unstructured has.